### PR TITLE
depot-tools: new formula

### PR DESCRIPTION
**Pull request by LinusU**
Depot tools from Google is a collection of tools for dealing with Chromium development.

http://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools.html

### CODE REVIEW

**Comment by paulirish** [comment]
yes would love to have this in! +1

File: depot-tools.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  depends_on "repo"
+
+  # We're linking these tools with auto-update disabled. It's bad enough that
+  # all we know is that we're at the HEAD of the master branch, since the
+  # Chromium team isn't nice enough to tag releases for us.
+  def install
+    dst = prefix/"tools"
+    dst.mkpath unless dst.directory?
+    mv Dir.glob("*"), dst
```
Comment (dunn) [inline depot-tools.rb]: These three lines can be simplified to `(prefix/"tools").install Dir["*"]`

Comment (LinusU) [inline depot-tools.rb]: Fixed :ok_hand:

**Comment by dunn** [comment]
Oops, also a minor audit failure:
```
==> audit problems
homebrew/head-only/depot-tools:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as homebrew/head-only/depot-tools + desc. (currently 90)

**Comment by LinusU** [comment]
Should be fixed now :+1:

**Comment by dunn** [comment]
Merged in 222837423bd13bb9de69b73d6766679f83725783! Thanks, and sorry for the delay!

**Comment by LinusU** [comment]
No problem, thank you!

**Pull request closed without merging**